src/tscore/runroot.cc

```diff
@@ -57,9 +56,9 @@ get_yaml_path(const std::string &path, bool json)
 {
   std::string yaml_file;
   if (ink_file_is_directory(path.c_str())) {
-    yaml_file = Layout::relative_to(path, "runroot_path.yml");
+    yaml_file = Layout::relative_to(path, "runroot.yml");
```

Comment:
Can we fix this up so it can look for the existing runroot_path.yml file and the runroot.yml ( and runroot.yaml)

This change is breaking for people using this feature already. Also Trafficserver is using .yaml for all the YAML file we make